python/tutorials/09-persistent-matmul.py

```diff
@@ -558,13 +572,9 @@ def bench(K, dtype, tiles_per_update, reps=10):
         for _ in range(reps):
             matmul_tma_persistent(a, b)
             time.sleep(0.01)
-        flops_str = "flops8" if dtype == torch.float8_e4m3fn else "flops"
-        with proton.scope(
-                f"matmul_kernel_device_tma_persistent M={M}, N={N}, K={K}, tiles_per_update={tiles_per_update:02}",
```

Comment:
I was using the scope to make sure tiles_per_update appears in the proton report.

Reply:
Added back